subsys/bluetooth/services/fast_pair/fp_keys.c

```diff
@@ -184,7 +185,8 @@ static bool key_gen_account_key_check(const uint8_t *account_key, void *context)
 	return true;
 }
 
-static int key_gen_account_key(struct bt_conn *conn, struct fp_keys_keygen_params *keygen_params)
+static int key_gen_account_key(const struct bt_conn *conn,
```

Comment:
Actually, we lose const here. We are writing it to context variable and pass context variable as non const.

Reply:
The const affects content of `struct bt_conn` pointed by the pointer. After casting the `void *` to a proper structure (the one that contains the `const struct bt_conn *`), you still will not be able to modify the `struct bt_conn` through the pointer which is part of the structure.